src/wallet/wallet.cpp

```diff
@@ -2799,6 +2799,12 @@ bool CWallet::CreateTransaction(interfaces::Chain::Lock& locked_chain, const std
 
             // Get the fee rate to use effective values in coin selection
             CFeeRate nFeeRateNeeded = GetMinimumFeeRate(*this, coin_control, &feeCalc);
+            // Do not, ever, assume that it's fine to change the fee rate if the user has explicitly
+            // provided one
+            if (coin_control.m_feerate && nFeeRateNeeded > *coin_control.m_feerate) {
+                strFailReason = strprintf("Fee rate is lower than the minimum fee rate setting. Increase it to %s or modify -minrelaytxfee", nFeeRateNeeded.ToString());
```

Comment:
I don't think we should encourage user to modify the min relay fee in reply to issues the wallet or rpc encounters. Lowering your min fee won't change transaction broadcast, as all your peers will still send you the same feefilter.

Reply:
Yeah, I'm not sure what to say here, though. That *is* the only way to do what they're attempting to do.